nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -358,6 +364,9 @@ contract StakingEscrow is Issuer {
                 allLockedTokens = allLockedTokens.add(lockedTokens);
             }
         }
+        assembly {
+            mstore(activeStakers, resultIndex)
```

Comment:
What is this doing?

Reply:
It's storing the array length in the first memory address

Reply:
More of a question is: why `mstore` in assembly instead of addressing `activeStakers` as an array? Optimization?

Reply:
There is no such thing like dynamic memory arrays in solidity, so this line is set real length for result memory array (removes zeros at the end of the array)

Reply:
Ah I see. Very cool!

Reply:
In this way better to return old name. I just thought there are not too many calls for policy creation right now